html-and-the-web/html-and-the-web_div-salad/index.html

```diff
@@ -15,49 +15,50 @@
 	which one was it again?-->
   <body>
 
-    <div>
+    <header>
       <div>
-        <div>Jane Doe</div>
+        <h1>Jane Doe</h1>
         <!--Which elements could be used to link to parts of the webpage,
         which appear below?-->
         <div id="nav">
-          <div>About Me</div>
-          <div>Contact</div>
-          <div>Skillset</div>
+          <a href="about-me">About Me</a>
+          <a href="contact">Contact</a>
+          <a href="skills">Skillset</a>
         </div>
       </div>
+    </header>
       <!--Looks like the following part is the central content of our webpage.-->
-      <div>
+      <main>
         <div id="about-me">
-          <div>About me</div>
-          <div>
+          <h2>About me</h2>
+          <p>
             Hi there! Welcome to my personal website. Here is gathered the most
             important information about me. If you want to work with me or get to
             know me, please don't hesitate to get in contact.
-          </div>
+          </p>
           <!--What element could be used to combine an image with a descriptive caption?-->
           <div>
             <img
-            src="https://source.unsplash.com/random/400x300?fruit"
+            src="https://www.sarikayagida.com.tr/tr/tekno_dosyalar/large/09_11_18_14266_322c245ad1920411f36afcb90ea1d1b9.jpg"
```

Comment:
its easier to download an image and then just drag an drop into the folder outline on the left side in VS Code.
and then add it to the website like this

```
<img
            src="code.jpg""
            alt="a funny picture"
/>
 ```

Reply:
Thank you! Do you mean drag and drop it in here: 
<img width="281" alt="image" src="https://github.com/user-attachments/assets/44584e34-96d1-4f4e-831c-193df65aeb2d">